rails/README.md

```diff
@@ -9,7 +9,7 @@ You can easily set up your laptop by running [kaishi](https://github.com/IcaliaL
 
 ## App creation
 
-We have developed a [script](https://github.com/IcaliaLabs/railsAppCustomGenerator) to help you start your rails apps with the bare bones that will accelerate your development process.
+We have developed a [script](https://github.com/IcaliaLabs/railsAppCustomGenerator) to help you start your rails apps with the barebones that will accelerate your development process.
```

Comment:
We have a gem for this, called [platter](https://github.com/IcaliaLabs/platter) I think we can update this!